docs/guides/Quick Start/02-product-management.md

```diff
@@ -0,0 +1,547 @@
+# Product Overview
+
+This section explains the handling of the product data structure.
+
+To start easy, below is a sample request to create a product with simple data from its complex API schema.
+
+## Create a new product with a simple payload
+
+A product has only a handful of required fields:
+
+* name [string]
+* productNumber [string]
+* taxId [string]
+* price [object]
+* stock [int]
+
+The smallest required payload for a product can therefore be as follows:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/product",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  },
+  "body": {
+    "name": "test",
+    "productNumber": "random",
+    "stock": 10,
+    "taxId": "a5da76b447db4d0aba62e6512dadf45b",
+    "price": [
+        {
+            "currencyId" : "b7d2554b0ce847cd82f3ac9bd1c0dfca", 
+            "gross": 15, 
+            "net": 10, 
+            "linked" : false
+        }
+    ]
+      }
+    }
+  }
+```
+
+```json json_schema
+{
+  "type": "object",
+  "description": "Parameters for product creation",
+  "properties": {
+    "name": {
+      "description": "Name of the product",
+      "type": "string"
+    },
+    "productNumber": {
+      "description": "Any random number given to product",
+      "type": "string"
+    },
+    "stock": {
+      "description": "Availability of stock",
+      "type": "string"
+    },
+    "taxId": {
+      "description": "ID of [tax](../../../adminapi.json/components/schemas/Tax)",
+      "type": "string"
+    },
+    "price": []
+      "currencyId": {
+      "description": "ID of [currency](../../../adminapi.json/components/schemas/Currency)",
+      "type": "string"
+      },
+      "gross": {
+      "description": "Gross price of a product",
+      "type": "string"
+      },
+      "net": {
+      "description": "Net price of a product",
+      "type": "string"
+      },
+      "linked": {
+      "description": "If set to true, net price is automatically calculated based on gross price and stored tax rate",
+      "type": "boolean"
+      }
+  }
+}
+```
+
+The `Productfeatureset` is a default template that extends from `productEntity` to insert all information for a new product.
```

Comment:
What is that `Productfeatureset` you are referring to here? - Probably the schema above, but it's not very visible for the reader. What you can do is add a `title` attribute to the schema definition. But you should point out, that this is no actual entity that exists in Shopware. - See my suggestion above.

I would also say it's not technically correct to say it "extends" from the `ProductEntity` (start entities with uppercase, please).